lib/ecto/migration/runner.ex

```diff
@@ -164,6 +165,10 @@ defmodule Ecto.Migration.Runner do
     log_and_execute_ddl(repo, log, {:create, index})
   end
 
+  defp execute_in_direction(repo, :backward, log, {:create, %Extension{} = extension}) do
+    log_and_execute_ddl(repo, log, {:drop, extension})
+  end
```

Comment:
I think we should be able to handle the extension with the generic clause here, that dispatches to `reverse/1`.